### DISCUSSION TITLE
Bug Report, Final Boss in Barotrauma's shield won't drop after Destroying the 3 pylons

### DISCUSSION THREAD

**Discussion by FatalMoment27** [FakeFishGames/Barotrauma#13932] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

After Destroying the 3 pylons during the final mission in Barotrauma the Boss's shield doesn't drop making the boss 100% invincible.  In previous patches of the game the shield would drop no problem, but after Blood in the Water patch it all of a sudden won't and I have replayed the mission several different times with the same result.

### Reproduction steps

The First 2 parts of the final mission run with no issues, its only when you arive to the final mission do I encounter this issue.  I would love to tell you how this occured because if I could I would have probably fixied it already.  I didnt have a problem with this mission until after the Blood in the Water patch went live.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
No crash reports, just the final boss becomes Invincible due too it's shield not lowering after the pylons are destroyed.
```

**Comment by BlackMoons** [comment]
You should post your campaign difficulty and if its been customized. Mainly the non-crew vitality multiplier has caused issues like this before, so is yours set to 1.0 or something else?

**Comment by Olance321** [comment]
Hello, have you resolved this issue? I'm having the same problem now

**Reply by Regalis11** [reply]
As BlackMoons suggested above, this issue could potentially be caused by some kind of customized campaign settings (or perhaps mods). Would it be possible to send us the save file so we could see if we can reproduce the issue there? Note that you need to .zip the file first, since GitHub doesn't support the .save file extension. Thanks and sorry for the inconvenience!

**Reply by Olance321** [reply]
I sent you an email. Can you check it?

**Reply by Regalis11** [reply]
Thanks! I unfortunately can't find an email, where exactly did you send it? Would it be instead possible to attach the save file in this thread in GitHub?

**Reply by Olance321** [reply]
[12312313.zip](https://github.com/user-attachments/files/18602445/12312313.zip)


Sorry. This is a saved file.

**Reply by Regalis11** [reply]
Thank you! I gave the save file a test, but unfortunately couldn't find a way to reproduce the issue. The shield seems to drop as expected after I destroy the pylons with the sub's turrets. I wonder if there could be some other factors affecting this - did you use some kind of console commands during the round for example? `repairitems` for example repairs every item in the level, including the pylons, which will interfere with the progress of the mission.

**Reply by Olance321** [reply]
I think there's a conflict element among the subscribed modes. I've never used console commands from start to finish.
Sad news. Maybe if I rebooted the game, the shield would have dropped.. Thanks for your response.

**Comment by Olance321** [comment]
Hello, I am sending you the save file, hope for good news. Do I need a list
of subscribed mods as well?


2025년 1월 30일 (목) 오전 3:00, Joonas Rikkonen ***@***.***>님이 작성:

**Reply by Regalis11** [reply]
Thanks! A list of mods won't be necessary, the save file includes information about which mods were enabled.

**Comment by jcx1991911** [comment]
A mod called "More stack" can cause this issue iirc

**Reply by Olance321** [reply]
Thank you so much. I was able to watch the ending after removing that mod. Thank you so much.. I really wanted to see the ending.
